src/config/countries-partial.php

```diff
@@ -17,11 +17,13 @@
         'diallingCode' => 47,
         'locale' => 'no',
         'language' => 'NOR',
+        'extension' => 'no',
```

Comment:
I think `tld` might be a slightly better option here?

Reply:
Yeah totally my bad.